modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/components/FieldSelectModalContent.scss

```diff
@@ -0,0 +1,35 @@
+@import 'atlas-variables';
+@import 'cadmin-variables';
+
+html#{$cadmin-selector} .cadmin .field-select-modal {
+	.selection-mode-multiple {
+		.custom-control-label {
+			line-height: 24px;
+		}
+
+		.field-label {
+			margin-left: 4px;
+		}
+
+		.treeview-group .autofit-col:has(> .custom-checkbox) {
+			width: 100%;
+		}
+
+		.treeview-link .custom-checkbox label {
+			display: block;
+			width: 100%;
+		}
+	}
+
+	.selection-mode-single .treeview-link {
+		color: $cadmin-gray-900;
+
+		.field-label {
+			padding: 3px;
+		}
+	}
+
+	.treeview-link {
+		margin: 0;
+	}
```

Comment:
This removed the two pixel inactive area on bottom and top of the item that @herrardo discovered.